README.md

```diff
@@ -28,13 +28,13 @@ href="NEWS.md">Release&nbsp;Notes</a>]</h3>
 Labwc stands for Lab Wayland Compositor, where lab can mean any of the
 following:
 
-- Lightweight And Box-inspired
+- lightweight and box-inspired
```

Comment:
I have a feeling `box-inspired` isn't that great. Better could be `*box-inspired` or even spelling out some of those. Personally I've only ever used Openbox so can't really comment about the others.

Reply:
Any thoughts @johanmalm ?